src/styles/style.css

```diff
@@ -0,0 +1,238 @@
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #eee;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+  animation: translateY(-20px);
+}
+
+.calendar__day:nth-child(1)::before {
+  content: "1";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(2)::before {
+  content: "2";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(3)::before {
+  content: "3";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(4)::before {
+  content: "4";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(5)::before {
+  content: "5";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(6)::before {
+  content: "6";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(7)::before {
+  content: "7";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(8)::before {
+  content: "8";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(9)::before {
+  content: "9";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(10)::before {
+  content: "10";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(11)::before {
+  content: "11";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(12)::before {
+  content: "12";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(13)::before {
+  content: "13";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(14)::before {
+  content: "14";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(15)::before {
+  content: "15";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(16)::before {
+  content: "16";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(17)::before {
+  content: "17";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(18)::before {
+  content: "18";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(19)::before {
+  content: "19";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(20)::before {
+  content: "20";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(21)::before {
+  content: "21";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(22)::before {
+  content: "22";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(23)::before {
+  content: "23";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(24)::before {
+  content: "24";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(25)::before {
+  content: "25";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(26)::before {
+  content: "26";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(27)::before {
+  content: "27";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(28)::before {
+  content: "28";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(29)::before {
+  content: "29";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(30)::before {
+  content: "30";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(31)::before {
+  content: "31";
+  font-size: 30px;
+}
```

Comment:
The content for `.calendar__day::before` is repetitive for each day and only the number changes. You can optimize this by using CSS counters to generate the content automatically. This will make your code DRY (Don't Repeat Yourself) and much easier to maintain. You can set up the counter in the `.calendar` class and increment it for each `.calendar__day`. For example:

.calendar {
  counter-reset: day;
}

.calendar__day::before {
  counter-increment: day;
  content: counter(day);
  font-size: 30px;
}